### ISSUE TITLE
tensorflow-js converter unable to convert 3D max pooling layers

### ISSUE THREAD

**Issue by AKhazane** [tensorflow/tfjs#1035]
Hello! I'm trying to convert a keras model to tensorflow.js (via tensorflow-js converter) and I'm running into an error with translating MaxPooling3D layers from my keras model into a corresponding 3D pooling layer in JS. After reading through the documentation, there doesn't appear to be a supported MaxPooling3D layer in the JS library. 

I did notice that a `tf.pool` function exists in the JS library which supports N-D pooling. Are there any immediate plans to either create an explicit 3D max pooling layer in tensorflow js, or perhaps modify the tensorflow-js converter to reference the `tf.pool` function while translating a N-D pooling layer from another library? Or maybe something else that would fix this issue?

Thanks!

**Comment by syt123450** [comment]
Hi @caisq ,  I sent several PRs to add the avgPool3d op, maxPool3d op, averagePooling3d layer, maxPooling3d layer. Could you please take a look at these PRs? Would appreciate hearing your thoughts on these PRs, thanks!

* Add maxPool3d and avgPool3d ops with CPU and WebGL kernel implementation, support forward prediction and gradient [tensorflow/tfjs-core#1778](https://github.com/tensorflow/tfjs-core/pull/1778)
* Make nodejs kernel support  avgPool3d & maxPool3d [tensorflow/tfjs-node#256](https://github.com/tensorflow/tfjs-node/pull/256)
* Add averagePooling3d layer & maxPooling3d layer [tensorflow/tfjs-layers#555](https://github.com/tensorflow/tfjs-layers/pull/555)
* Add avgPool3d & maxPool3d ops to graph model executor [tensorflow/tfjs-converter#375](https://github.com/tensorflow/tfjs-converter/pull/375)

**Comment by rthadur** [comment]
related PR #[375](https://github.com/tensorflow/tfjs-converter/pull/375) has been merged , will be closing this issue.
